### PR TITLE
Adding dry run error hook

### PR DESCRIPTION
**Pull request by pr4bh4sh**

### CODE REVIEW

File: lib/parallel_cucumber/main.rb [hunk 1]
```diff
@@ -30,8 +30,12 @@ def run
         exit(1)
       end
 
-      all_tests = Helper::Cucumber.selected_tests(@options[:cucumber_options], @options[:cucumber_args])
-
+      begin
+        all_tests = Helper::Cucumber.selected_tests(@options[:cucumber_options], @options[:cucumber_args])
+      rescue StandardError => e
+        Hooks.fire_on_dry_run_error({ failure_reason: e.message, backtrace: e.backtrace, cucumber_options: @options[:cucumber_options], cucumber_args:  @options[:cucumber_args] })
```
Comment (fr0l) [inline lib/parallel_cucumber/main.rb:36]: Please pass the whole error object to the block. otherwise we'll be missing `e.class` and `e.cause`

Comment (fr0l) [reply lib/parallel_cucumber/main.rb:36]: also please could you omit brackets ? 

        Hooks.fire_on_dry_run_error(error: e, cucumber_options: @options[:cucumber_options], cucumber_args:  @options[:cucumber_args])

Comment (fr0l) [reply lib/parallel_cucumber/main.rb:36]: not sure though how and why needed to use `cucumber_options` and `cucumber_args`

**Review by fr0l — CHANGES_REQUESTED** [review]
Please could you take a look at the comments

File: lib/parallel_cucumber/main.rb (hunk 1 above)
Comment (pr4bh4sh) [reply lib/parallel_cucumber/main.rb]: Params are removed now, only error is passed

**Review by fr0l — APPROVED** [review]

**Pull request merged**